assets/bin/grub2-editenv

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# This is a workaround for TuneD running in the container not to emit
```

Comment:
```suggestion
# This is a workaround to prevent TuneD running in the container from reporting
```
nit, I find the wording a bit awkward here.

Reply:
Thank you David.  Fixed.